examples/webserver/index.ts

```diff
@@ -4,7 +4,7 @@ import * as os from "@pulumi/openstack";
 // OpenStack installation you're using.
 const instance = new os.compute.Instance("test", {
     flavorName: "s1-2",
-    imageName: "Ubuntu 18.04",
+    imageName: "Ubuntu 22.04",
```

Comment:
According to https://wiki.ubuntu.com/Releases, looks like `18.04` is no longer supported as of June '23. `22.04` should be good for a while until '27.